_posts/2018-09-18-volunteer-opportunities.md

```diff
@@ -13,11 +13,13 @@ We have a wide variety of volunteer slots available! If you’d like to sign up
 
-Session runners are the shepherds of the speaker room. Your job starts in the Green Room (Santa Fe 3) fifteen minutes before each talk where you’ll meet your speaker and help them get ready by testing their laptop and briefing them on what to expect. Five minutes before their talk, you’ll walk with them to their session room, get them water, and help them get their laptop set up. You’ll hand the speaker off to the session chair, and then *relax and enjoy the talk*. Your job is done until 15 minutes before the next talk!
+Session Managers are part of the DjangoCon US organizing team, and they will oversee the chairs and runners. Their responsibilities are to get the printed introduction sheets to the chairs and ensure everyone knows where to go. If a session chair or runner is unavailable, the manager will serve as backup, and be responsibile for marshaling resources to ensure the talks proceed on schedule.
 
-Session chairs are the moderators of the session. You’ll arrive in the speaker room 10 minutes before the talk starts. Meet your speaker and find out a few things: how do you pronounce your name? Do you want to take questions? How do you want us to count down time? At showtime, you’ll introduce the speaker to the audience, keeping it simple — their name, where they’re from, and talk title. During the talk, watch the clock and let the speaker know how much time they have left before questions (show signs at the -15, -10 and -5 mark). Mediate questions (if any), and let speaker and attendees know when it’s time to wrap it all up. After that, help the speaker disconnect the microphone and make any announcements.
+Session runners meet speakers outside the room where they'll be speaking fifteen minutes before each talk. You’ll meet your speaker and help them get ready by testing their laptop and briefing them on what to expect. Five minutes before their talk, you’ll walk with them to the stage, get them water, and help them get their laptop set up. You’ll hand the speaker off to the session chair, and then *relax and enjoy the talk*. Your job is done until 15 minutes before the next talk!
+
+Session chairs are the moderators of the session. You’ll arrive at the session 10 minutes before the talk starts. Meet your speaker and find out a few things: how do you pronounce your name? Do you want to take questions? How do you want us to count down time? At showtime, you’ll introduce the speaker to the audience, keeping it simple — their name, where they’re from, and talk title. During the talk, watch the clock and let the speaker know how much time they have left before questions (show signs at the -15, -10 and -5 mark). Mediate questions (if any), and let speaker and attendees know when it’s time to wrap it all up. After that, help the speaker disconnect the microphone and make any announcements.
```

Comment:
Remove "where they're from", we are simplifying this to talk title and name of presenter only